tldw_Server_API/app/services/admin_bundle_service.py

```
@@ -80,8 +83,7 @@ def _get_schema_versions() -> dict[str, int | None]:
     """Collect current schema versions from DB modules (lazy imports)."""
     versions: dict[str, int | None] = {}
     try:
-        from tldw_Server_API.app.core.DB_Management.Media_DB_v2 import MediaDatabase
-        versions["media"] = int(MediaDatabase._CURRENT_SCHEMA_VERSION)
+        versions["media"] = get_current_media_schema_version()
     except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)